db/db_impl.cc

```diff
@@ -5058,6 +5058,26 @@ void DBImpl::MaybeFlushColumnFamilies() {
     return;
   }
 
```

Comment:
> let's flush its memtables anyway (btw, why?).

We flush the memtables we can now and the remaining tables later. It makes sense intuitively to me but maybe its not actually optimal behavior? It also reduces the chance of another transaction coming in and 'preparing' into a memtable that we could have flushed now will be unable to flush in the future. This guarantees that we are making progress on releasing the log.